submissions/adeyemiJuwon/todo/src/components/Todos.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react';
+import TodoList from './TodoItem';
+
+export default class Todos extends Component {
+  render() {
+    return (
+      <>
```

Comment:
Remove this

Reply:
just a short rep of <React.Fragment> instead for me to use div without any className